tools/bindsnoop.py

```diff
@@ -418,9 +432,45 @@ def opts2str(bitfield):
         bit *= 2
     return str_options.encode()
 
+def opts2array(bitfield):
+    options = []
+    bit = 1
+    for opt in "FTNRr":
+        if bitfield & bit:
+            options.append(opt)
```

Comment:
To enable users to parse the output without iterating on the array, could it be defined differently?
```
{
  comm: "nc",
  opts: {
    "F": false,
    "T": true,
    "N": false,
    "R": false,
    "r": false
  }
}
```